claims_p2.go

```diff
@@ -202,7 +254,16 @@ func (c P2Claims) ToJSON() ([]byte, error) { //nolint:gocritic
 }
 
 func (c P2Claims) ToUnvalidatedJSON() ([]byte, error) { //nolint:gocritic
+	var scs ISwComponents
+	if c.SwComponents != nil && c.SwComponents.IsEmpty() {
```

Comment:
I think, the same questions apply here as on line 212

Reply:
yeah it was the same idea, and not needed heire either as `SwComponents` are not `omitempty`.